### PR TITLE
Rel 518428 remove rsapi from group helpers

### PR DESCRIPTION
**Pull request by aarongilbert123**

### CODE REVIEW

File: Source/Relativity.Test.Helpers/Relativity.Test.Helpers.NUnit.Integration/GroupHelpers/GroupHelpersIntegrationTests.cs [hunk 1]
```diff
@@ -55,29 +60,28 @@ public void CreateGroupAndDeleteGroupTest()
...
-
-					WriteResultSet<kCura.Relativity.Client.DTOs.Group> deleteResultSet = rsapiClient.Repositories.Group.Delete(groupArtifactIds);
-					if (!deleteResultSet.Success)
+						ObjectType = new ObjectTypeRef
+						{
+							ArtifactTypeID = 3
+						},
+						Condition = $"'Name' == '{groupName}'"
+					};
+					Services.Objects.DataContracts.QueryResult queryResult = objectManager.QueryAsync(-1, clientQueryRequest, 0, 10).GetAwaiter().GetResult();
```
Comment (jr-martin) [inline Source/Relativity.Test.Helpers/Relativity.Test.Helpers.NUnit.Integration/GroupHelpers/GroupHelpersIntegrationTests.cs:75]: According to Thor's comments on a PR of mine, we should be setting .ConfigureAwait(false) when not using async operations. here's an example: https://github.com/relativitydev/relativity-test-helpers/blob/develop/Source/Relativity.Test.Helpers/Relativity.Test.Helpers/ArtifactHelpers/FieldHelper.cs#L61

Comment (aarongilbert123) [reply Source/Relativity.Test.Helpers/Relativity.Test.Helpers.NUnit.Integration/GroupHelpers/GroupHelpersIntegrationTests.cs:75]: Done

**Review by jake-martin — APPROVED** [review]

**Review by TommyBigNose — CHANGES_REQUESTED** [review]
Change log

File: Source/Relativity.Test.Helpers/Relativity.Test.Helpers.NUnit.Integration/GroupHelpers/GroupHelpersIntegrationTests.cs [hunk 2]
```diff
@@ -3,11 +3,15 @@
...
-using kCura.Relativity.Client;
 using kCura.Relativity.Client.DTOs;
 using NUnit.Framework;
 using Relativity.API;
+using Relativity.Services.Interfaces.Group;
+using Relativity.Services.Objects;
+using Relativity.Services.Objects.DataContracts;
+using Relativity.Services.ServiceProxy;
 using Relativity.Test.Helpers.SharedTestHelpers;
+using QueryResult = kCura.Relativity.Client.QueryResult;
```
Comment (calimetikcura) [inline Source/Relativity.Test.Helpers/Relativity.Test.Helpers.NUnit.Integration/GroupHelpers/GroupHelpersIntegrationTests.cs:14]: - [ ] Remove RSAPI using statement

File: Source/Relativity.Test.Helpers/Relativity.Test.Helpers.NUnit.Integration/GroupHelpers/GroupHelpersIntegrationTests.cs [hunk 3]
```diff
@@ -55,29 +60,28 @@ public void CreateGroupAndDeleteGroupTest()
...
+					};
+					Services.Objects.DataContracts.QueryResult queryResult = objectManager.QueryAsync(-1, clientQueryRequest, 0, 10).GetAwaiter().GetResult();
+					if (queryResult.TotalCount > 0)
 					{
-						throw new Exception("Failed to Delete Test Groups");
+						foreach (var obj in queryResult.Objects)
+						{
+							using (IGroupManager groupManager = serviceFactory.CreateProxy<IGroupManager>())
+							{
+								groupManager.DeleteAsync(obj.ArtifactID).Wait();
```
Comment (calimetikcura) [inline Source/Relativity.Test.Helpers/Relativity.Test.Helpers.NUnit.Integration/GroupHelpers/GroupHelpersIntegrationTests.cs:82]: - [ ] Use ConfigureAwait(false)

**Comment by aarongilbert123** [comment]
> Change log

**Review by TommyBigNose — APPROVED** [review]

File: Source/Relativity.Test.Helpers/Relativity.Test.Helpers/GroupHelpers/GroupHelper.cs [hunk 4]
```diff
@@ -1,104 +1,179 @@
...
+			}
+		}
+
+		public static bool DeleteGroup(Services.ServiceProxy.ServiceFactory serviceFactory, int artifactId)
+		{
+			try
+			{
+				using (Services.Interfaces.Group.IGroupManager groupManager = serviceFactory.CreateProxy<Services.Interfaces.Group.IGroupManager>())
+				{
+					groupManager.DeleteAsync(artifactId).Wait();
```
Comment (calimetikcura) [inline Source/Relativity.Test.Helpers/Relativity.Test.Helpers/GroupHelpers/GroupHelper.cs:77]: - [ ] Use ConfigureAwait(false)

**Review by calimetikcura — CHANGES_REQUESTED** [review]

**Pull request merged**